src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs

```diff
@@ -165,6 +165,28 @@ public static IEnumerable<object[]> GetDataForJson()
             });
         }
 
+        public static IEnumerable<object[]> GetDataForStu3ToR4()
+        {
+            var data = new List<string[]>
+            {
+                new[] { @"CapabilityStatement" },
+                new[] { @"CodeSystem" },
+                new[] { @"Observation" },
+                new[] { @"OperationDefinition" },
+                new[] { @"OperationOutcome" },
+                new[] { @"Parameters" },
+                new[] { @"Patient" },
+                new[] { @"StructureDefinition" },
+                new[] { @"ValueSet" },
+            };
+            return data.Select(item => new[]
+            {
+                item[0],
+                Path.Join(Constants.SampleDataDirectory, "STU3", item[0] + ".json"),
+                Path.Join(Constants.ExpectedDataFolder, "Json", "ExpectedR4Resource", item[0] + ".json"),
```

Comment:
I think we can maintain consistency here. It seems that the sample data about Stu3 has its own folder `STU3` instead of `Json`, while the files about R4 are placed in the folder `Json`.

Reply:
Stu3 to R4 will has its own folder both in templates folder,  sample data folder and expected result folder.